modules/microshift-uninstalling-lvms-csi-snapshot.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * microshift_storage/microshift-storage-plugin-overview.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-uninstalling-lvms-csi-snapshot_{context}"]
+== Uninstalling the CSI driver and CSI snapshot implementation
+
+To uninstall the installed CSI driver and CSI snapshot implementation, use the following procedure.
+
+.Procedure
+
+. To uninstall the CSI driver implementation, run all the following commands:
+* To delete the `vg-manager`, run the following command:
```

Comment:
```suggestion
. Delete the `vg-manager` by running the following command:
```
etc; these should be numbered individually and omit "to"; active voice only

Reply:
removed "To".
I made this as a bulleted list as this numbering is not working as expected and there is no specific order in executing these commands.